packages/types/api/replaySqsMessages.d.ts

```diff
@@ -0,0 +1,4 @@
+export interface ReplaySqsMessagesPayload {
+  queueName: string,
+  type?: string,
```

Comment:
This isn't part of the payload anymore

Reply:
[`030ed97` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/030ed97f35a8892454a2f46e229a87b2bb447203#diff-72236dd0282583674134919c52d1834ec33487006430075c7ec21c9908b37182L3)